doc/guidelines/frontend-ui-usability.md

```diff
@@ -0,0 +1,195 @@
+
+## Terminology
+
+### Usability
+
+> Extent to which a system, product or service can be used by specified users to achieve specified goals with effectiveness, efficiency and satisfaction in a specified context of use.
+
+### User Interface (UI)
+
+> Set of all the components of an interactive system that provide information and controls for the user to accomplish specific tasks with the interactive system.
+
+### User Experience (UX)
+
+> Combination of user's perceptions and responses that result from the use and/or anticipated use of a system, product or service.
+
+## Guidelines
+
+This guide should help improve the UI (User Interface) and usability of the PermaplanT web application going forward.  
+The following guidelines are:
+
+- on the one hand a mix of general UI and usability best practices and
+- on the other hand design suggestions tailored to PermaplanT.
+
+### Text
+
+- **Size**
+
+  - `<h1>` Headings : `text-2xl`
+  - `<h2>` Subheadings: `text-xl`
+  - `<h3>` Subheadings (currently not used/needed): `text-lg`
+  - Navigation items: `text-lg`
+  - Running text: `text-sm`
+  - Form fields and labels: `text-sm`
+
+- **Family**
+
+  - PermaplanT is using Tailwind's default cross-browser sans-serif stack everywhere.
+  - If a different family is to be used in the future, extend the theme with a new `theme.fontFamily` section in our Tailwind config file (`tailwind.config.js`).
+
+- **Alignment**
+
+  - Always left align everything.
+  - Exceptions:
+    - Modals: center headings
+    - Guided tour: center highlighted actions
+
+- **Spacing / margin**
+
+  - `<h1>` Headings: `mb-8`
+  - `<h2>` Subheadings: `mb-2`
+  - Cards (like on our Map overview): `mb-4`
+  - Form fields (like on the Seeds page): `mb-8`
+  - Labels of form fields: `mb-2`
+
+- **Letter spacing**
+
+  - Always go with the default value, i.e. `tracking-normal`.
+  - In case letter spacing changes are considered in the future:
+    - Headings: use negative values for the bigger headings h1-h3, i.e. `tracking-tighter` or `tracking-tight`
+    - Running text: `tracking-wide` or `tracking-wider`
+
+### Colors
+
+PermaplanT uses following **color palettes**:
+
+- **Neutral** (PermaplanT: gray): **dominant** color of the design, e.g. used for background, text and labels.
+- **Primary** (PermaplanT: asparagus green): PermaplanT's **main brand** color, e.g. used for call-to-action buttons, checkmarks, highlighting focused form input fields, hovering over icons, highlighting active/enabled icons in the toolbox.
+- **Secondary** (PermaplanT: sea blue): PermaplanT's **second brand** color, e.g. used for highlighting selected plants on the map, currently focused icons, spinners, action-texts in the guided tour and bigger UI elements like the transparency bars to have some contrast to the primary color.
+- Tailwind's default color palette for **red** is e.g. used to render validation errors in forms, form field asterisks, buttons for destructive actions.
```

Comment:
maybe we could mention the exact color that should be user for the different use cases. For example you mentioned below that the astrisks should be red-500, but i would excpect this here in the color section.